src/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
```

Comment:
Is this file a duplicate of client/src/index.jsx but using jsx? If the other one is a transpiled file from this one, it should be added to the gitignore

Reply:
I kep one that was without jsx but have since removed it to make it cleaner. Should I resubmit the PR or would it make sense to have this duplicate file removal reflected on the next PR?

Reply:
I'll approve it on this pull request, just take it out for the next one.